contracts/scripts/test/smokeTest.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+trap "exit" INT TERM ERR
+trap "kill 0" EXIT
+nodeWaitTimeout=60
+
+main()  
+{
+    nodeOutput=$(mktemp "${TMPDIR:-/tmp/}$(basename 0).XXX")
+    yarn run node:fork &> $nodeOutput &
```

Comment:
Don't we need to set BLOCK_NUMBER here? Though see my comment below regarding not necessarily needing this script.

Reply:
Yeah from my tests it seems to run fine without it.

Reply:
What about PROVIDER_URL, that's another env var that should be required.
Maybe add a quick check in the script and exit or show a warning in the console if either BLOCK_NUMBER or PROVIDER_URL are missing.

Reply:
Good point thanks, adding those checks